server/views/visualisations.py

```diff
@@ -34,13 +35,14 @@ def get_all_metrix():
     return db.get_all_metrix()
 
 
-def pca(data):
+def reduce_dimensionality(data, reducer, normalize=True):
     data = data.dropna()
     data.pop('_id')
     data.pop('session_id')
     user_ids = data.pop('user_id')
-    sklearn_pca = PCA(n_components=2)
-    reduced = pd.DataFrame(sklearn_pca.fit_transform(StandardScaler().fit_transform(data)))
+    if normalize:
```

Comment:
Ako tak pozeram, toto tu je vzdy `True`, naco to tu je?

Reply:
Keď som to robil som si nebol istý či sa má škálovať aj pri t-SNE, tak som tam dal tú možnosť neškálovať. So škálovaním to ale dávalo krajšie grafy, tak nakoniec sa škáluje 🤷‍♂️. Môžeme dať preč ale nemyslím že to niečomu vadí.